### PR TITLE
Added const model with validation constraints

### PR DESCRIPTION
**Pull request by DaniilPoiarkov**
Added const model to storage validation constraints and replaced existing validations with this model

### CODE REVIEW

File: frontend/src/app/components/register-page/register-page.component.ts [hunk 1]
```diff
@@ -23,19 +24,22 @@ export class RegisterPageComponent implements OnInit {
   faGithub = faGithub;
   faGoogle = faGoogle;
 
-  constructor() { 
+  constructor(
+    public validationConstraints = ValidationConstraints,
+  ) {
```
Comment (Setup1337) [inline frontend/src/app/components/register-page/register-page.component.ts]: Why you use constructor for this if `ValidationConstraints` is constant?

Comment (DaniilPoiarkov) [reply frontend/src/app/components/register-page/register-page.component.ts]: Habit:D

File: frontend/src/entity-models/const-resources/validation-constraints.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const ValidationConstraints = {
+    minLengthName: 4,
+    minLengthPassword: 8,
+    minLengthEmail: 8
+};
```
Comment (ismv) [inline frontend/src/entity-models/const-resources/validation-constraints.ts]: eof fix pls

Comment (DaniilPoiarkov) [reply frontend/src/entity-models/const-resources/validation-constraints.ts]: > eof fix pls

Done

**Review by Its-OP — CHANGES_REQUESTED** [review]
Maybe it's better to call the model "ValidationConstants" instead of "ValidationConstraints" ? Because these values do not really limit anything, Validators do.

**Review by Its-OP — APPROVED** [review]

**Review by ismv — APPROVED** [review]

**Pull request merged**